src/workerd/api/actor-state.h

```diff
@@ -348,11 +350,41 @@ class DurableObjectState: public jsg::Object {
   jsg::Promise<jsg::Value> blockConcurrencyWhile(jsg::Lock& js,
       jsg::Function<jsg::Promise<jsg::Value>()> callback);
 
+  void acceptWebSocket(jsg::Ref<WebSocket> ws, kj::Array<kj::String> tags);
+  // TODO(now): We get a bunch of errors when using an Array of StringPtr's -- figure this out.
```

Comment:
The errors would be expected. Since this function is mapped out to JavaScript, there is no inbound type mapping for kj::StringPtr, only kj::String.

Reply:
Ah, that makes sense, I was confused by this comment in `jsg.h`
```
// Additionally, the following types are always supported:
// ... other types
// - C++ kj::String, kj::StringPtr <-> JS String
```